<h1>
    Lab Sheet 14 Solutions
</h1>

<h3>
    Lab Sheet 14 Problem 1
</h3>

Riders on a newly designed amusement park ride are seated
in cars arranged around a circle of radius $R=10\,m$. Once the ride
begins, each of the cars revolves around the centre of the circle and
a hoisting arm of length $A=13\,m$ tilts the entire circle into the
vertical plane. The $x$-, $y$- and $z$-positions of one particular car are
given (in metres) by the functions:
\begin{align*}
  x(\theta,\psi)&=\left(A+R\cos\theta\right)\cos\psi \\
  y(\theta)&=R\sin\theta \\
  z(\theta,\psi)&=\left(A+R\cos\theta\right)\sin\psi
\end{align*}
where $\theta$ is the angle in the plane of the circular arrangement
of cars between the specified car and point opposite the hoisting arm,
and $\psi$ is the tilt angle that the plane of the circle makes with
the horizontal.

Assume the angle of revolution and the tilt angle are
given (in radians) respectively by:
\begin{align*}
  \theta(t)&=80\left(1-e^t\right) \\
  \psi(t)&=\frac{\pi{}t}{2(t+1)}
\end{align*}
for $t$ measured in minutes. Find
$\frac{\partial{}x}{\partial{}t},\frac{\partial{}y}{\partial{}t},\frac{\partial{}z}{\partial{}t}$
and use them to determine both the velocity vector $\vec{v}$ and the
speed for the specified car at time $t=1$ minute after the ride
begins. 

<img src="./diagrams/lceheavb.png" width="40%" height="40%">

Recall that the multivariable chain rule tells us that, if $w=f(x,y)$
has continuous first-order partial derivatives and $x=g(t)$ and
$y=h(t)$ are differentiable functions, then $w$ is a differentiable
function of $t$, and
    \begin{equation}
      \label{eq:ylwkabbx}
      \frac{dw}{dt}=\frac{\partial{}w}{\partial{}x}\cdot\frac{dx}{dt}+\frac{\partial{}w}{\partial{}y}\cdot\frac{dy}{dt}\notag
    \end{equation}

<img src="./diagrams/lceheavb1.png" width="80%" height="80%">

<h3>
    Lab Sheet 14 Problem 2
</h3>

Find the area between the two curves
    \begin{align*}
      r&=2a+a\sin{}2\theta \\
      r&=2a+a\cos{}2\theta
    \end{align*}

<img src="./diagrams/uqcnlnvn.png" width="40%" height="40%">

In [1]:
tht1=*(0.5,atan(1))

0.39269908169872414

In [2]:
tht2=-(tht1,/(pi,2))

-1.1780972450961724

$$
A=2\int_{\theta_2}^{\theta_1}\left(2a+a\sin(2\theta)\right)^2\,d\theta
$$

In [19]:
using QuadGK

In [33]:
integral1,err=quadgk(x->*(4,+(1,sin(*(2,x)),*(0.25,^(sin(*(2,x)),2)))),tht2,tht1,rtol=1e-8)

(4.240156345830845, 2.532196674565057e-12)

In [34]:
*(2,36,integral1) # for example if a=6

305.29125689982084

<h3>
    Lab Sheet 14 Problem 3
</h3>

Find the area of the inner loop of the limacon with
polar equation
\begin{equation}
  \label{eq:nrnrbkac}
  r=9\cos\theta-8\notag
\end{equation}

<img src="./diagrams/hyondzel-desmos.png" width="40%" height="40%">

In [31]:
tht3=acos(/(8,9))

0.47588224966041665

In [35]:
integral2,err=quadgk(x->^(-(*(9,cos(x)),8),2),0,tht3,rtol=1e-8)

(0.2524275821016009, 5.551115123125783e-17)

Note that at $\theta_3=\arccos\left(\frac{8}{9}\right)$ the radius becomes negative. When we graph polar coordinates with a negative radius, we add $\pi$ to the angle, so
$$
(-r,\theta)=(r,\theta+\pi)
$$
for positive $r$. The area of the inner loop is
$$
A=\int_{0}^{\theta_3}\left(9\cos\theta-8\right)^2\,d\theta\approx{}0.2524
$$